source/declarations.tex

```diff
@@ -7105,6 +7114,11 @@
 Each $\tcode{v}_i$ is the name of an lvalue of type $\tcode{T}_i$
 that refers to the object bound to $\tcode{r}_i$;
 the referenced type is $\tcode{T}_i$.
+The initialization of \tcode{e} and
+any conversion of \tcode{e} considered as a decision variable\iref{stmt.pre}
```

Comment:
\tcode{e} -> \exposid{e}

Reply:
fixed

Reply:
looks like you only fixed the one on line 7118 - please fix the one on line 7117 also